src/style.css

```diff
@@ -1 +1,77 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+  font-style: normal;
+}
+
+.page {
+  color: #3d4e61;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 300;
+}
+
+.page__form {
+  margin-top: 20px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here you have only added margin-top which is good, just ensure you don't add bottom margin to the same element elsewhere to maintain consistency.